core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -315,18 +369,15 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.editTopic = async function(topicName) {
-    await waitFor.elementToBeClickable(
-      topicsTabButton, 'Unable to click on topics tab.');
-    var topicElements = await _getTopicElements(topicName);
-    if (topicElements.length === 0) {
-      throw new Error('Could not find topic tile with name ' + topicName);
-    }
-    await waitFor.elementToBeClickable(
-      topicElements[0], 'Unable to click on topic: ' + topicName);
-    await topicElements[0].click();
-    await waitFor.pageToFullyLoad();
-    await waitFor.invisibilityOf(
-      topicElements[0], 'Topic editor page takes too long to load.');
+    await waitFor.visibilityOf(topicsTable,
+      'Topic table taking too long to appear.');
+
+    Promise.all(await topicNames.map(async(topic, index) => {
```

Comment:
@Showtim3 I don't think you need the `Promise.all` either since `Promise.all` just returns a promise that returns/rejects based on the promises it's passed. Docs here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
Yeah I checked locally, it's working.
Done